src/renderer/CategoricalStackedDistributionlRenderer.ts

```diff
@@ -22,7 +22,7 @@ export default class CategoricalStackedDistributionlRenderer implements ICellRen
   createGroupDOM(col: ICategoricalColumn & Column): IDOMGroupRenderer {
     const colors = col.categoryColors;
     const labels = col.categoryLabels;
-    const bins = col.categories.map((c, i) => `<div style="background-color: ${colors[i]}" title="${labels[i]}: 0" data-cat="${c}">${labels[i]}</div>`).join('');
+    const bins = col.categories.map((c, i) => `<div style="background-color: ${colors[i]}; --text-color: ${adaptTextColorToBgColor(colors[i])}" title="${labels[i]}: 0" data-cat="${c}">${labels[i]}</div>`).join('');
```

Comment:
isn't the labels already part of the content -> and later on the `data-cat` will be overlayed?

Reply:
Yes, the label is available as element content. However, as you found out it's not possible to shift it. That's why I made the element text invisible with `color: transparent` and set the color for the pseudo-element using this CSS custom property (or black as default).

Reply:
why do you add then the label at all, if you hide it anyhow?

Reply:
Right. I changed it now and use a `span` element instead, which makes the pseudo-element and CSS custom properties obsolete.